azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpointsProperties.java

```diff
@@ -13,7 +13,7 @@
 import java.util.Map;
 
 @Configuration
-@ConfigurationProperties("azureservice")
+@ConfigurationProperties("azure.service")
```

Comment:
seems there's no instruction in document about how to set this property?

Reply:
Yes, I try to find that word in repo including README. But only below change appears. We may enhance this in short future.